### PR TITLE
Components work with updated lambda functions

### PR DESCRIPTION
**Pull request by ratthompson**

### CODE REVIEW

File: batches/BatchService.ts [hunk 1]
```diff
@@ -1,27 +1,41 @@
 import axios from 'axios';
 import Batch from './Batch';
 
+interface trainerBatchResp {
+	batches: Batch[];
+	validYears: [];
```
Comment (Tngtater9) [inline batches/BatchService.ts]: validYears array needs a type is it number[] or string[]

Comment (ratthompson) [reply batches/BatchService.ts]: have now added the number type

**Review by Tngtater9 — COMMENTED** [review]
need auth headers then good to go

**Comment by Tngtater9** [comment]
Make sure you handle your code smells

**Pull request merged**